### PR TITLE
Add complex `i` constant function

### PR DESCRIPTION
**Pull request by hauleth**

### CODE REVIEW

File: src/complex.rs [hunk 1]
```diff
@@ -59,6 +59,14 @@ impl<T: Clone + Num> Complex<T> {
     }
 }
 
+impl<T: Clone + Num> Complex<T> {
+    /// Returns `i`
+    #[inline]
+    pub fn i() -> Complex<T> {
+        Self::new(T::zero(), T::one())
+    }
+}
```
Comment (cuviper) [inline src/complex.rs]: Why did you make this a separate `impl` block from the one above?
I'd also be a little more descriptive, like "Returns the imaginary unit."

Comment (hauleth) [inline src/complex.rs]: Accidentally as I firstly wrote that for types that are `Clone + Zero + One`, not `Clone + Num`.

Comment (cuviper) [inline src/complex.rs]: @homu r+

**Comment by cuviper** [comment]
@homu r+

**Pull request merged**